examples/Example19/main.cpp

```diff
@@ -254,9 +254,18 @@ int main(int argc, char *argv[])
   scoringPerVolume.energyDeposit      = energyDeposit;
   GlobalScoring globalScoring;
 
+  const bool testEm3 = gdml_file.find("estEm3") != std::string::npos;
```

Comment:
Matching for a specific name is very, very ugly. Can we make this more general, please?

Reply:
Like I said on mattermost, I would just pass the values via command line arguments, like: `example19 -gunpos -220,0,0 -gundir 1,0,0`. This way we don't need to hardcode anything.

Reply:
Better to put it in the readme. The defaults are correct for CMS.

Reply:
Added to readme.